texar/torch/modules/pretrained/pretrained_gpt2.py

```diff
@@ -44,6 +44,11 @@ class PretrainedGPT2Mixin(PretrainedMixin, ABC):
     by `Radford et al.` from OpenAI. It is a unidirectional Transformer model
     pre-trained using the vanilla language modeling objective on a large corpus.
 
```

Comment:
Could you also fix the docstring for `_init_from_checkpoint`? The `pretrained_model_name` argument is undocumented.